examples/diff/index.html

```diff
@@ -0,0 +1,160 @@
+<!doctype html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <!-- Core extensions CSS from Contentful. -->
+  <link rel="stylesheet"
+        href="//contentful.github.io/ui-extensions-sdk/cf-widget-api.css">
+  <!-- Styles for added / removed text -->
+  <style>
+    .diff-text .added {
+      font-weight: bold;
+      color: #5cb85c;
+    }
+
+    .diff-text .removed {
+      text-decoration: line-through;
+      font-weight: bold;
+      color: #950B02;
+    }
+  </style>
+</head>
+<body>
+
+<!-- Extension DOM Elements -->
+<div class="cf-form-field">
+  <input type="text" class="cf-form-input draft-text">
+  <div class="cf-form-hint">
+    <span class="diff-text"></span>
+  </div>
+</div>
+
+<!-- Contenful UI Extensions SDK API -->
+<script src="//contentful.github.io/ui-extensions-sdk/cf-widget-api.js"></script>
+<!-- Diff Lib -->
+<script src="//cdnjs.cloudflare.com/ajax/libs/jsdiff/2.2.2/diff.min.js"></script>
+
+<!-- Diffing UI Extension -->
+<script>
+
+  function diffExtension(extensionsApi) {
+
+    extensionsApi.window.startAutoResizer();
+
+    // DOM elements.
+    var draftTextEl = document.querySelector('.draft-text');
+    var diffTextEl = document.querySelector('.diff-text');
+
+    // Disable the input as we need to wait for an async call to load the published entry.
+    draftTextEl.disabled = true;
+
+    var entry = extensionsApi.entry;
+    var field = extensionsApi.field;
+
+    // Currently published version of entry.
+    var publishedVersion = extensionsApi.entry.getSys().publishedVersion;
+    var publishedValue;
+
+    // Current locale
+    var fieldLocale = extensionsApi.field.locale;
+    var fieldId = extensionsApi.field.id;
+
+    // Callbacks for changes on the field or the entry.sys object.
+    // Detach functions could be used to clean up the callbacks.
+    var detachValueChanged = field.onValueChanged(onValueChanged);
```

Comment:
You could invoke the detach methods when this iframe is unloaded. So something like: 
```js
window.addEventListener('beforeunload', function () {
  detachValueChanged();
  detachSysChanged();
});
```